backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/archive/api/IArchiveBL.java

```diff
@@ -69,4 +71,9 @@ public interface IArchiveBL extends ISingletonService
 	Optional<I_AD_Archive> getLastArchiveRecord(@NonNull TableRecordReference reference);
 
 	Optional<Resource> getLastArchiveBinaryData(@NonNull TableRecordReference reference);
+
+	I_AD_Archive retrieveArchive(@NonNull ArchiveId archiveId);
```

Comment:
call it getById

Reply:
getRecordById because getById already existins and return AdArchive